conf.d/hibernate.conf

```diff
@@ -0,0 +1,13 @@
+NAME="Hibernate Validator Aggregator"
+DESCRIPTION="Aggregator of the Hibernate Validator modules."
+HOME="http://hibernate.org/validator"
+LICENSE="ASL 2.0"
+COPYRIGHT=""
+COPYRIGHT_NOTE=''
+VERSION="6.0.18"
+ARTIFACT0="hibernate-validator.jar"
+ARTIFACT1="hibernate-validator.pom"
+ARTIFACT2="hibernate-validator-parent.pom"
+VERIFIER=""
+VERIFY_DATE=""
+OVIRT_VERSIONS="4.5"
```

Comment:
also here about the verification part, anything else needed other than checking engine builds with this?

Reply:
If you have already everything set for the test please go ahead :-) I'm building a missing dep on el9.

Reply:
Build passes. RPM build will fail later on at
```
Exception: Following dependencies were not resolved and requires cannot be generated. Either remove the dependencies from pom.xml or add proper packages to BuildRequires:
org.infinispan:infinispan-core:9.4.18.Final required by org.ovirt.engine.core:utils
org.jboss.modules:jboss-modules:1.1.1.GA required by org.ovirt.engine.core:common
```
going to add them in next PRs. If the PR looks good to you let's merge and move to next library.

Reply:
seems to work